🐛 Error When Running 'npm install'

#### Describe your issue

When running 'npm install' with `platform-folders` already installed I sometimes get this error:

````
npm ERR! path /bpmn-studio/node_modules/platform-folders
npm ERR! code EISGIT
npm ERR! git /bpmn-studio/node_modules/platform-folders: Appears to be a git repo or submodule.
npm ERR! git /bpmn-studio/node_modules/platform-folders
npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.
````

Seems to be about this https://github.com/kingjan1999/platform-folders/issues/7 issue, which was fixed in a newer version of `platform-folders`

#### How can others reproduce the issue?

- Run `npm install`
- Run `npm install` again

#### Possible solution

Update `platform-folders` to `0.2.7`.

#### Issue checklist

Please check the boxes in this list after submitting your Issue:

- [x] I've checked if this issue already exists
- [x] I've included all the information that i think is relevant
- [x] I've added logs and/or screenshots (if applicable)
- [x] I've mentioned PRs and issues that relate to this one

--- a/package.json
+++ b/package.json
@@ -48,7 +48,7 @@
     "@process-engine/timers.repository.sequelize": "2.0.2",
     "addict-ioc": "2.3.7",
     "loggerhythm": "3.0.3",
-    "platform-folders": "0.2.6",
+    "platform-folders": "0.2.7",
     "sqlite3": "4.0.2"
   }
 }

